src/plugins/intel_gpu/src/graph/graph_optimizer/post_optimize_weights.cpp

```diff
@@ -125,9 +125,9 @@ void post_optimize_weights::optimize_weights(T& node, program& p) {
                 if (node.type() == lstm_seq::type_id()) {
                     program_node& prev_node = node.get_dependency(i);
                     if (i == 5) {
-                        add_lstm_bias_reorder(prev_node.id(), weights_reorder_params, p, prev_node, node);
+                        add_lstm_bias_reorder(prev_node.id() + node.id(), weights_reorder_params, p, prev_node, node);
```

Comment:
nit: Isn't a separator like '_' needed between the two IDs?

Reply:
Updated! Thanks!